po/tr.po

```diff
@@ -0,0 +1,178 @@
+
+#: data/io.github.lainsce.Khronos.desktop.in:11
+msgid "Plot;Planning;Time Tracking;"
+msgstr ""
+
+#: data/io.github.lainsce.Khronos.gschema.xml:5
+msgid "Delay between notifications"
+msgstr ""
+
+#: data/io.github.lainsce.Khronos.gschema.xml:6
+msgid "The delay between two notifications, default is 30min. in seconds."
+msgstr "Bildirimler arası süre varsayılan olarak 30 dakikadır"
+
+#: data/ui/main_window.ui:6
+msgid "Export Logs as CSV…"
+msgstr "Kayıtları bir CSV dosyasına aktar"
+
+#: data/ui/main_window.ui:10
+msgid "Import Logs from CSV…"
+msgstr "Kayıtları CSV dosyasından içeri aktar"
+
+#: data/ui/main_window.ui:16
+msgid "About Khronos"
+msgstr "Khronos Hakkında"
+
+#: data/ui/main_window.ui:53
+msgid "Timer"
+msgstr "Zamanlayıcı"
+
+#: data/ui/main_window.ui:77
+msgid "Log name…"
+msgstr "Kayıt adı..."
+
+#: data/ui/main_window.ui:96
+msgid "Adds a log to the list"
+msgstr "Listeye bir kayıt ekler"
+
+#: data/ui/main_window.ui:106 src/MainWindow.vala:161
+msgid "Starts the timer for a log"
+msgstr "Kayıt için zamanlayıcıyı başlatır"
+
+#: data/ui/main_window.ui:122
+msgid "Logs"
+msgstr "Kayıtlar"
+
+#: data/ui/main_window.ui:148
+msgid "Removes all logs from the list"
+msgstr "Listedeki tüm kayıtları siler"
+
+#: data/ui/main_window.ui:180
+msgid "No Logs"
+msgstr "Kayıt Yok"
+
+#: data/ui/main_window.ui:188
+msgid "Add a log on Timer tab and check later."
+msgstr ""
+
+#: src/MainWindow.vala:153
+msgid "Stops the timer for a log"
+msgstr "Kaydı tamamlamak için zamanlayıcıyı durdur"
+
+#: src/MainWindow.vala:186
+msgid "Empty the Logs List?"
+msgstr "Kayıt listesi boşaltılsın mı?"
+
+#: src/MainWindow.vala:187
+msgid ""
+"Emptying this list means all the logs in it will be permanently lost with no "
+"recovery."
+msgstr ""
+"Kayıtları listesini boşalttığınızda tüm kayıtlarınız kalıcı olarak "
+"silinecektir."
+
+#: src/MainWindow.vala:189
+msgid "Cancel"
+msgstr "İptal"
+
+#: src/MainWindow.vala:190
+msgid "Empty List"
+msgstr "Boş Liste"
+
+#: src/MainWindow.vala:307
+msgid "Go rest for a while before continuing."
+msgstr "Devam etmeden önce biraz dinlen"
+
+#: src/MainWindow.vala:316
+msgid "Maybe grab a snack before continuing."
+msgstr "Belki devam etmeden ufak bir atıştırmalık almalısın"
+
+#: src/MainWindow.vala:325
+msgid "Perhaps go get some coffee or tea before continuing."
+msgstr "Devam etmeden önce belki kahve veya çaya alabilirsin"
+
+#: src/MainWindow.vala:334
+msgid "That's a big task. Let's rest a bit before continuing."
+msgstr "Ne uzun bir çalışmaydı, devam etmeden önce hadi biraz mola verelim."
+
+#: src/MainWindow.vala:343
+msgid "Amazing work! But please rest a bit before continuing."
+msgstr "Harika bir çalışma! ama devam etmeden önce lütfen biraz dinlen"
+
+#: src/MainWindow.vala:387
+msgid "Track each task's time in a simple inobtrusive way."
+msgstr "Her görev için harcadığınız zamanı basitçe takip edin."
+
+#: src/MainWindow.vala:393
+msgid "translator-credits"
+msgstr "çevirmenler ve geliştiriciler"
```

Comment:
This is not for translation, this is where you write your own name and email like that:
Sinan AYDOĞAN <sinanaydogan@hotmail.com.tr>